tests/chains/MultiversX/SignerTests.cpp

```diff
@@ -416,4 +415,54 @@ TEST(MultiversXSigner, SignESDTNFTTransfer) {
     ASSERT_EQ(expectedEncoded, encoded);
 }
 
+TEST(MultiversXSigner, SignGenericActionWithGuardian) {
+    auto input = Proto::SigningInput();
+    auto privateKey = PrivateKey(parse_hex(ALICE_SEED_HEX));
+    input.set_private_key(privateKey.bytes.data(), privateKey.bytes.size());
+
+    input.mutable_generic_action()->mutable_accounts()->set_sender_nonce(42);
+    input.mutable_generic_action()->mutable_accounts()->set_sender(ALICE_BECH32);
+    input.mutable_generic_action()->mutable_accounts()->set_receiver(BOB_BECH32);
+    input.mutable_generic_action()->mutable_accounts()->set_guardian(CAROL_BECH32);
+    input.mutable_generic_action()->set_value("1000000000000000000");
+    input.mutable_generic_action()->set_data("");
+    input.mutable_generic_action()->set_version(2);
+    input.mutable_generic_action()->set_options(TransactionOptions::Guarded);
+    input.set_gas_price(1000000000);
+    input.set_gas_limit(100000);
+    input.set_chain_id("1");
+
+    auto output = Signer::sign(input);
+    auto signature = output.signature();
+    auto encoded = output.encoded();
+    auto expectedSignature = "dae30e5cddb4a1f050009f939ce2c90843770870f9e6c77366be07e5cd7b3ebfdda38cd45d04e9070037d57761b6a68cee697e6043057f9dc565a4d0e632480d";
+    auto expectedEncoded =
+        (boost::format(R"({"nonce":42,"value":"1000000000000000000","receiver":"%1%","sender":"%2%","gasPrice":1000000000,"gasLimit":100000,"chainID":"1","version":2,"signature":"%3%","options":2,"guardian":"%4%"})") % BOB_BECH32 % ALICE_BECH32 % expectedSignature % CAROL_BECH32).str();
```

Comment:
Let's avoid usage of boost please, we try to remove it

Reply:
Fixed, `boost::format` is not used anymore.